gt-elasticsearch/src/main/java/mil/nga/giat/data/elasticsearch/ElasticParserUtil.java

```diff
@@ -315,9 +312,11 @@ private Coordinate createCoordinate(final List<Object> posList) {
 
             if (entry == null) {
                 entry = source.get(name);
+                readField(entry, Collections.EMPTY_LIST, values);;
```

Comment:
Can you change entry to `source.get(name)` directly in `readField` call and then you can make `entry` final above? Also remove extra trailing semi-colon.